backend/attendance/views.py

```diff
@@ -29,7 +38,14 @@ def list_next_lessons_with_details(self, request):
         lessons_list_serialized = LessonWithDetailsSerializer(queryset, many=True)
 
         return Response(lessons_list_serialized.data, status=status.HTTP_200_OK)
-    
+
+    def list_mobile_lessons_with_details(self, request):
+        queryset = Lesson.objects.all()
```

Comment:
Não precisa declarar o queryset aqui se ele for igual ao que tá na classe (linha 23)